test/unit/connection_string.spec.test.ts

```diff
@@ -3,7 +3,8 @@ import { executeUriValidationTest } from '../tools/uri_spec_runner';
 
 const skipTests = [
   // TODO(NODE-3914): Fix; note that wtimeoutms will be deprecated via DRIVERS-555 (NODE-3078)
-  'Deprecated (or unknown) options are ignored if replacement exists'
+  'Deprecated (or unknown) options are ignored if replacement exists',
+  'Colon in a key value pair'
```

Comment:
Can we create a ticket with the `spec-compliance` label about the failing 'Colon in a key value pair' test?

Reply:
There's no ticket as we are not going to fix that. Node's URI class cannot handle it and it was deemed acceptable that this specific case is provided as a MongoClient option only. See the TOKEN_RESOURCE section in MONGODC-OIDC auth mech propeties: https://github.com/mongodb/specifications/blob/master/source/auth/auth.md#machine-authentication-flow

Reply:
Add added a comment to the test to note this.